gradio/blocks.py

```diff
@@ -1583,6 +1601,27 @@ def reverse(text):
                 "mode": self.mode,
             }
             utils.launch_analytics(data)
+            additional_data = {
```

Comment:
adding some additional analytics here. Such as a list of event triggers and blocks components. Although what components are inputs vs outputs is lost here. Open to suggestions for more analytics to track

Reply:
Yeah combining both makes sense. I can separate blocks components into list of inputs and outputs, should be easy enough.

So we'll have `gradio/initiated`, `gradio/launched`, `gradio/error`, and `gradio/integrations` as topics

Reply:
Lists of components would make it hard to query imo. It would be nice if we can easily query things like "how often is the ColorPicker component used"? What if we have a `gradio/components` topic that stores:

* component type
* mode: input, output, both, or neither  

that both Interface and Blocks use?

Reply:
Would that require a separate request per component in the demo?

Im reading this as storing each component in a separate row

Reply:
Yup it would require a separate request per component as far as I understand, but if we can fire these off asynchronously in the background in `launch()`, there should be no performance overhead

Reply:
Thinking about this some more, that would be way too many network requests. Ignore my suggestion, let's stick with the topics you've proposed @dawoodkhan82! Though we should handle the fact that Blocks can have components that are both inputs and outputs, or neither.